### PR TITLE
Added `feature_dcos_storage_enabled` flag to configuration parameters

### PR DESCRIPTION
**Pull request by asridharan**
for an enterprise install.

## Urgency
- [ ] Blocker 
- [ ] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

---
This change is [Reviewable](https://reviewable.io/reviews/mesosphere/dcos-docs-site/664)

### CODE REVIEW

**Comment by asridharan** [comment]
@jieyu could you take a look at this docs PR.

File: pages/1.11/installing/ent/custom/configuration/configuration-parameters/index.md [hunk 1]
```diff
@@ -57,6 +57,12 @@ This topic provides configuration parameters available for [DC/OS Enterprise](ht
 | [dcos_l4lb_enable_ipv6](#dcos_l4lb_enable_ipv6)        | A boolean that indicates if layer 4 load balancing is available for IPv6 networks. |
 |[dcos_ucr_default_bridge_subnet](#dcos-ucr-default-bridge-subnet) |IPv4 subnet allocated to the `mesos-bridge` CNI network for UCR bridge-mode networking. | 
 
+# Storage 
+
+| Parameter                    | Description                                                                                                                                                       |
+|------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [feature_dcos_storage_enabled](#feature-dcos-storage-enabled-enterprise)           | A flag that enables [CSI support](https://github.com/container-storage-interface/spec) on the Mesos agents. By default set to `false`. This feature flag needs to be turned on to use the [Data Storage Service (DSS)](/services/beta-storage)|
```
Comment (jieyu) [inline pages/1.11/installing/ent/custom/configuration/configuration-parameters/index.md]: DC/OS Storage Service

Comment (jieyu) [inline pages/1.11/installing/ent/custom/configuration/configuration-parameters/index.md]: I'd say: `A flag, if set, will enable advanced storage features in DC/OS, including Mesos CSI support, pre-loaded device plugins, etc. This feature flag needs to be turned on to use the [DC/OS Storage Service (DSS)](/services/beta-storage)`

File: pages/1.11/installing/ent/custom/configuration/configuration-parameters/index.md [hunk 2]
```diff
@@ -576,6 +582,14 @@ The bridge-mode networking for UCR is identical to bridge mode networking for Do
 
 The only constraint in selecting an IPv4 subnet for `dcos_ucr_default_bridge_subnet` is that the subnet should not be used on the network to which the agents are connected. In other words, this subnet should be routeable from only within an agent.
 
+[enterprise]
+### feature_dcos_storage_enabled
+[/enterprise]
+
+Enables CSI support in the Mesos agents. Currently the only CSI plugin that is supported is the LVM plugin. The plugin allows Mesos to expose local volumes as resources to frameworks.
```
Comment (jieyu) [inline pages/1.11/installing/ent/custom/configuration/configuration-parameters/index.md]: Similar to above, i'd just mention that this flags will enable advanced storage support in DC/OS. I won't mention lvm at all here.

**Comment by asridharan** [comment]
@jieyu can you take another look?

File: pages/1.11/installing/ent/custom/configuration/configuration-parameters/index.md [hunk 3]
```diff
@@ -576,6 +582,14 @@ The bridge-mode networking for UCR is identical to bridge mode networking for Do
 
 The only constraint in selecting an IPv4 subnet for `dcos_ucr_default_bridge_subnet` is that the subnet should not be used on the network to which the agents are connected. In other words, this subnet should be routeable from only within an agent.
 
+[enterprise]
+### feature_dcos_storage_enabled
+[/enterprise]
+
+Enables advanced storage features in Mesos including [CSI](https://github.com/container-storage-interface/spec) support, and support for pre-installed CSI device plugins. 
+* `feature_dcos_storage_enabled: 'false'` Disables CSI support in the Mesos agent. This is the default value.
```
Comment (jieyu) [inline pages/1.11/installing/ent/custom/configuration/configuration-parameters/index.md]: Please adjust those as well. s/Mesos/DC/OS/

File: pages/1.11/installing/ent/custom/configuration/configuration-parameters/index.md [hunk 4]
```diff
@@ -576,6 +582,14 @@ The bridge-mode networking for UCR is identical to bridge mode networking for Do
 
 The only constraint in selecting an IPv4 subnet for `dcos_ucr_default_bridge_subnet` is that the subnet should not be used on the network to which the agents are connected. In other words, this subnet should be routeable from only within an agent.
 
+[enterprise]
+### feature_dcos_storage_enabled
+[/enterprise]
+
+Enables advanced storage features in Mesos including [CSI](https://github.com/container-storage-interface/spec) support, and support for pre-installed CSI device plugins.
```
Comment (jieyu) [inline pages/1.11/installing/ent/custom/configuration/configuration-parameters/index.md]: s/Mesos/DC/OS/

Comment (jieyu) [reply pages/1.11/installing/ent/custom/configuration/configuration-parameters/index.md]: Enables advanced storage features in DC/OS including Mesos CSI support, and pre-installed CSI device plugins.

File: pages/1.11/installing/ent/custom/configuration/configuration-parameters/index.md [hunk 5]
```diff
@@ -57,6 +57,12 @@ This topic provides configuration parameters available for [DC/OS Enterprise](ht
 | [dcos_l4lb_enable_ipv6](#dcos_l4lb_enable_ipv6)        | A boolean that indicates if layer 4 load balancing is available for IPv6 networks. |
 |[dcos_ucr_default_bridge_subnet](#dcos-ucr-default-bridge-subnet) |IPv4 subnet allocated to the `mesos-bridge` CNI network for UCR bridge-mode networking. | 
 
+# Storage 
+
+| Parameter                    | Description                                                                                                                                                       |
+|------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [feature_dcos_storage_enabled](#feature-dcos-storage-enabled-enterprise)           | A flag, if set, will enable advanced storage features in DC/OS, including Mesos CSI support and pre-installed device plugins. This feature flag needs to be turned on to use the [DC/OS Storage Service (DSS)](/services/beta-storage)|
```
Comment (jieyu) [inline pages/1.11/installing/ent/custom/configuration/configuration-parameters/index.md]: pre-installed CSI device plugins to be consistent with below.

Comment (jieyu) [reply pages/1.11/installing/ent/custom/configuration/configuration-parameters/index.md]: have a link for CSI too.

**Comment by asridharan** [comment]
@jieyu  can you take a look?

**Pull request merged**